stdlib/http/src/main/ballerina/src/http/cookie/cookie.bal

```diff
@@ -58,62 +58,62 @@ public type Cookie object {
         return true;
     }
 
-    // Returns true if the attributes of the cookie are in the correct format; false otherwise.
+    // Returns true if the attributes of the cookie are in the correct format or else error is returned.
     public function isValid() returns boolean | error {
-        error invalidCookieError;
-        var temp = self.name;
-        if (temp is string) {
-            temp = temp.trim();
-            if (temp == "") {
-                invalidCookieError = error("Invalid name");
-                return invalidCookieError;
+        CookieHandlingError err;
+        var name = self.name;
+        if (name is string) {
+            name = name.trim();
+            if (name == "") {
+                err = error(COOKIE_HANDLING_ERROR, message = "Invalid name");
```

Comment:
Without saying 'invalid name', can't we specifically say that name cannot be empty? Because here we are checking whether the name is an empty string.